third_party/move/move-vm/runtime/src/frame.rs

```diff
@@ -213,6 +216,7 @@ impl Frame {
             ty_builder,
             locals,
             function,
+            _guard: guard,
```

Comment:
nit: field should not have leading _ in the name?

Reply:
The field is never read so it needs the underscore.